File: 1066.py
par = 0
impar = 0
pos = 0
neg = 0

for x in range(0,5):
    n = int(input())
    if n % 2 == 0:
        par += 1
    else:
        impar += 1
    if (n > 0 and n != 0):
        pos += 1
    if (n < 0 and n != 0):
        neg += 1

print(par, 'valor(es) par(es)')
print(impar, 'valor(es) impar(es)')
print(pos, 'valor(es) positivo(s)')
print(neg, 'valor(es) negativo(s)')

File: 1066.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let par = 0
let impar = 0
let pos = 0
let neg = 0

for (i = 0;  i < 5; i++){
    let n = parseInt(lines.shift());
    if (n%2 == 0){
        par += 1
    }       
    else{
        impar += 1
    }
        
    if (n > 0 && n != 0){
        pos += 1
    }
        
    if (n < 0 && n != 0){
        neg += 1
    }
        
}

console.log(par + ' valor(es) par(es)');
console.log(impar + ' valor(es) impar(es)');
console.log(pos + ' valor(es) positivo(s)');
console.log(neg + ' valor(es) negativo(s)');

